Doc/reference/datamodel.rst

```diff
@@ -1501,7 +1501,9 @@ Basic customization
 
    Called by built-in function :func:`hash` and for operations on members of
    hashed collections including :class:`set`, :class:`frozenset`, and
-   :class:`dict`.  :meth:`__hash__` should return an integer. The only required
+   :class:`dict`.
+   
+   :meth:`__hash__` should return an integer. The only required
```

Comment:
```suggestion
   :class:`dict`. ``__hash__()`` should return an integer. The only required
```

I don't think we need a line break here -- surely we can just take out the link, since it's only linking to the paragraph we're already on?

Reply:
I agree with Raymond's comment on the issue, to use normal English.
```suggestion
   :class:`dict`.  The__hash__ method should return an integer. The only required
```